index.html

```diff
@@ -0,0 +1,349 @@
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+    <img src="./assets/images/Gourp1-desktop.png" alt="Image in the first page">
+  </section>
+  <section class="cards">
+    <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
+    </div>
+    <img class="main-image" src="assets/images/Placeholder.png" alt="main image">
+
+    <div class="card-1">
+      <h2 class="multi-story">Multi-Post Stories</h2>
+      <p class="p21">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="skills">
+        <li class="css-1 button">css</li>
+        <li class="skill button">html</li>
+        <li class="skill button">bootstrap</li>
+        <li class="skill button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-1 button" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2, button" href="">See Project</a>
+
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class=" desk-prof-art">Profesional Art Printing Data</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
```

Comment:
The left and right spacing of your project section should be equal to make it more like the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=aZU1XLEXM0PwGPUU-0) 😉

yours
<img width="960" alt="yoursecttt" src="https://user-images.githubusercontent.com/64643995/220945685-28f182d3-fcb7-4c05-959a-2e9a5dcc7009.png">

design
<img width="516" alt="design sect" src="https://user-images.githubusercontent.com/64643995/220946242-2fe92cf7-48b7-43b5-a95b-42fbfda05811.png">